x-pack/legacy/plugins/siem/public/pages/hosts/details/utils.ts

```diff
@@ -40,10 +43,13 @@ export const getBreadcrumbs = (params: RouteSpyState, search: string[]): Breadcr
     ];
   }
   if (params.tabName != null) {
+    const tabName = get('tabName', params);
```

Comment:
nit - maybe we can refactor this to use `useParams` for `react-router-dom`

Reply:
It complained about using hook here in this library, so I leaved it